project3/index.html

```diff
@@ -0,0 +1,379 @@
+    <form action="">
+        <section>
+            <h2> IDENTITY </h2>
+
+            <label for="real-name">Real Name</label>
+            <input id="real-name" type="text" placeholder="" required>
+
+            <label for="alias">Alias</label>
+            <input id="alias" type="text" placeholder="" required>
+
+            <label for="email">Email</label>
+            <input id="email" type="email" placeholder="" required>
+        </section>
+        <section class="age">
+            <h2> PERSONAL CHARACTERISTICS </h2>
+            <label for="age">Age</label>
+            <input id="age" type="number" placeholder="" required>
+        </section>
+        <section>
+            <fieldset class="check">
+                <legend>Category</legend>
+                <p>
+                    Please select as many as apply to your origin and how you gain your powers.
+                </p>
+                <div>
+                    <input id="alien" type="checkbox">
+                    <label for="alien">Alien</label>
+                </div>
+                <div>
+                    <input id="cyborg" type="checkbox">
+                    <label for="cyborg">Cyborg</label>
+                </div>
+                <div>
+                    <input id="experiment" type="checkbox">
+                    <label for="experiment">Experiment</label>
+                </div>
+                <div>
+                    <input id="human" type="checkbox">
+                    <label for="human">Human</label>
+                </div>
+                <div>
+                    <input id="inhuman" type="checkbox">
+                    <label for="inhuman">Inhuman</label>
+                </div>
+                <div>
+                    <input id="mutant" type="checkbox">
+                    <label for="mutant">Mutant</label>
+                </div>
+
+            </fieldset>
+
+            <fieldset class="rad">
+                <legend>Gender</legend>
+                <div>
+                    <input name="gender" id="women" type="radio">
+                    <label for="women">A Women</label>
+                </div>
+                <div>
+                    <input name="gender" id="man" type="radio">
+                    <label for="man">A Man</label>
+                </div>
+                <div>
+                    <input name="gender" id="nonbinary" type="radio">
+                    <label for="nonbinary">Nonbinary</label>
+                </div>
+                <div>
+                    <input name="gender" id="genderqueer" type="radio">
+                    <label for="genderqueer">Genderqueer</label>
+                </div>
+                <div>
+                    <input name="gender" id="pefernottoanswer" type="radio">
+                    <label for="pefernottoanswer">Pefer not to answer</label>
+                </div>
+                <div>
+                    <span> <label for="pefernottoanswer">Pefer to Describe</label></span>
```

Comment:
This `label`'s `for` attribute doesn't match its `input`:
```suggestion
                    <span> <label for="pefertodescribe">Pefer to Describe</label></span>
```